pipelines/manual-release-pipeline.yml

```diff
@@ -254,7 +298,132 @@ jobs:
       KUBERNETES_CLUSTER: ((kubernetes-cluster-name))
       KUBERNETES_DEPLOYMENT_NAME: notify-processor
       KUBERNETES_SELECTOR: app=notify-processor
-      KUBERNETES_FILE_PATH: kubernetes-repo/microservices
+      KUBERNETES_FILE_PATH: kubernetes-repo/release/microservices
       KUBERNETES_FILE_PREFIX: notify-processor
       WAIT_UNTIL_AVAILABLE_TIMEOUT: 200s
-    input_mapping: {kubernetes-repo: census-rm-kubernetes-microservices-repo}
+    input_mapping: {kubernetes-repo: census-rm-kubernetes-release}
+
+- name: "Exception Manager"
+  serial: true
+  serial_groups: [sit-exception-manager]
+  plan:
+  - get: census-rm-kubernetes-release
+    trigger: true
+    passed: ["Trigger All"]
+  - get: census-rm-deploy
+  - get: every-minute
+    trigger: true
+    passed: ["Trigger All"]
+  - task: apply-service-and-deploy
+    file: census-rm-deploy/tasks/kubectl-apply-service-and-deploy-no-patch.yml
+    on_failure: *slack_failure_alert
+    params:
+      SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+      GCP_PROJECT_NAME: ((sit-gcp-project-name))
+      KUBERNETES_CLUSTER: ((sit-kubernetes-cluster-name))
```

Comment:
Shouldn't these just be using `gcp-project-name` and `kubernetes-cluster-name`, in line with all of the others?  Same for the other new ones that have been added below.  The serial groups don't matter so much but is there a reason why they're prefixed with `sit-`?